AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Requests/SendRequestStatusToGetASmokeAlarmHandlerShould.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using AllReady.Models;
+using Xunit;
+
+namespace AllReady.UnitTest.Areas.Admin.Features.Requests
+{
+    using System.Linq.Expressions;
+    using System.Threading.Tasks;
+
+    using AllReady.Areas.Admin.Features.Requests;
+    using AllReady.Hangfire.Jobs;
+
+    using global::Hangfire;
+    using global::Hangfire.Common;
+    using global::Hangfire.States;
+
+    using Moq;
+
+    public class SendRequestStatusToGetASmokeAlarmHandlerShould : InMemoryContextTest
+    {
+        private readonly Guid requestId;
+
+        public SendRequestStatusToGetASmokeAlarmHandlerShould()
+        {
+            this.requestId = Guid.NewGuid();
+            this.Context.Requests.Add(new Request
+            {
+                RequestId = this.requestId,
+                Source = RequestSource.Api
+            });
+            this.Context.SaveChanges();
+        }
+
+        [Fact]
+        public async Task NotQueueUpAnythingIfTheRequestStatusIsNotASupportedStatus()
```

Comment:
:+1: for testing this type of stuff. Lots of times, these types of tests are missed...